Source/WebCore/html/DOMTokenList.cpp

```diff
@@ -234,8 +234,14 @@ void DOMTokenList::updateTokensFromAttributeValue(StringView value)
         unsigned end = start + 1;
         while (end < value.length() && !isASCIIWhitespace(value[end]))
             ++end;
+        bool wholeAttributeIsSingleToken = !start && end == value.length();
+        if (wholeAttributeIsSingleToken) {
+            m_tokens.append(value);
+            addedTokens.add(WTFMove(value));
```

Comment:
We do not need to add it to `addedTokens`.

Reply:
oh, good point.